src/Common/src/Interop/Windows/kernel32/Interop.FormatMessage.cs

```diff
@@ -3,8 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Text;
 using System.Runtime.InteropServices;
+using System.Text;
```

Comment:
What I suggest doing with this file is: leave it unchanged. Make a new file next to it named `Interop.FormatMessage_SafeLibraryHandle.cs`. Into that put what you have here, ie., the FormatMessage that takes SafeLibraryHandle.
This will mean you can revert all the changes to the other C# projects that add SafeLibraryHandle and FreeLibrary.

Reply:
There is more refactoring needed (copy the helper GetMessage code into there from EventLog code, and eliminate `GetMessage(IntPtr moduleHandle` changing its only existing caller to use the SafeLibraryHandle one) but that can happen later. My suggestion above will make it simpler for now.